homeassistant/components/flo/device.py

```diff
@@ -7,6 +7,7 @@
 from aioflo.api import API
 from aioflo.errors import RequestError
 from async_timeout import timeout
+from requests.exceptions import JSONDecodeError
```

Comment:
This is not the correct error that is being used, according to the linked issue, it is using: `orjson.JSONDecodeError`.

As this integration has a test suite, let's make sure we add a test case for this.

../Frenck